g/godbusv5/LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2013, Georg Reinke (<guelfey at gmail dot com>), Google
```

Comment:
this needs to be the script license which is Apache 2.0 and not the package license

Reply:
Fixed license file